src/main/java/com/a2/backend/model/UserPrivacyDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.a2.backend.model;
+
+import com.a2.backend.constants.PrivacyConstant;
+import lombok.*;
+
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@ToString
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserPrivacyDTO {
+
+    @NotNull
+    private PrivacyConstant tagsPrivacy;
+
+    @NotNull
+    private PrivacyConstant languagesPrivacy;
+
+    @NotNull
+    private PrivacyConstant ownedProjectsPrivacy;
+
+    @NotNull
+    private PrivacyConstant collaboratedProjectsPrivacy;
```

Comment:
@TheAvengersInitiative/devs Idk if asking Front for an Enum is ok. Should these fields be strings and then the service converts them to enums? Right now, if front sends a string that is not "PUBLIC" or "PRIVATE" a 400 is returned with error message "Bad Request".

Reply:
Its alright. This enum particularly is pretty static, and has no reason to change in the near future, so it's ok like this.

For a case in which the enum might change a lot (think of languages, for example) you would also add and endpoint `/privacy/options` that returns all the values of the enum, as a way to inform the frontend of the options available.